### DISCUSSION TITLE
New ssh key

### DISCUSSION THREAD

**Discussion by Azul808** [community/community#51450] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I forgot my passphrase for my ssh key to my local machine so I can not clone repositories into my machine. I want to create a new SSH Key.  I know I need to delete the keys from Github but also form my local machine. When I command $ cd .ssh   I get the following 4  

id_ed25519
id_ed25519.pub
known_hosts
known_hosts.old

Do I need to delete all 4 of these or just the first 2? Please help.

**Comment by airtower-luna** [comment, marked as answer]
`id_ed25519` and `id_ed25519.pub` are the private and public key files for your ED25519 key. If you want to delete and replace that key, deleting those two is sufficient.

The `known_hosts` file records the public keys of servers you have already connected to, so you don't need to confirm if they are correct on each connection.

**Reply by Azul808** [reply]
Thank you so much, appreciate the help.

**Comment by MJBrachais00** [comment]
> `id_ed25519` and `id_ed25519.pub` are the private and public key files for your ED25519 key. If you want to delete and replace that key, deleting those two is sufficient.
> 
> 
> 
- > The `[known]()_hosts` file records the public keys of servers you have already connected to, so you don't need to confirm if they are correct on each connection.
